server/src/ClientToDBAPI/ClinicalDetailsRoute/mainRoute.ts

```diff
@@ -1,9 +1,58 @@
 import { Router, Request, Response } from 'express';
 
+import { graphqlRequest } from '../../GraphqlHTTPRequest';
+import { CREATE_ADDRESS, CREATE_INVESTIGATION, ADD_BACKGROUND_DESEASES, ADD_SYMPTOMS } from '../../DBService/ClinicalDetails/Mutation';
+
 const clinicalDetailsRoute = Router();
 
 clinicalDetailsRoute.get('/', (request: Request, response: Response) => {
     response.send('Hello from Clinical Details route');
-})
+});
+
+clinicalDetailsRoute.post('/saveClinicalDetails', (request: Request, response: Response) => {
+    
+    const requestAddress = request.body.input;
+
+    const requestInvestigation = request.body.investigation;
+    
+    graphqlRequest(CREATE_ADDRESS, { input: {
+        ...requestAddress
+    }}).then((result: any) => {
+        graphqlRequest(CREATE_INVESTIGATION, { investigation: {
+            epidemioligyNumber: requestInvestigation.epidemioligyNumber,
+            creator: requestInvestigation.creator,
+            lastUpdator: requestInvestigation.lastUpdator,
+            isolationAddress: result.data.insertAndGetAddressId.integer,
+            endTime: requestInvestigation.endTime,
+            handler: requestInvestigation.handler,
+            hospitalDepartment: requestInvestigation.hospitalDepartment,
+            hospital: requestInvestigation.hospital,
+            hospitalizationEndTime: requestInvestigation.hospitalizationEndTime,
+            hospitalizationStartTime: requestInvestigation.hospitalizationStartTime,
+            investigatingUnit: requestInvestigation.investigatingUnit,
+            investigatedPatientId: requestInvestigation.investigatedPatientId,
+            investigationStatus: requestInvestigation.investigationStatus,
+            isInIsolation: requestInvestigation.isInIsolation,
+            isIsolationProblem: requestInvestigation.isIsolationProblem,
+            isIsolationProblemMoreInfo: requestInvestigation.isIsolationProblemMoreInfo,
+            isolationStartTime: requestInvestigation.isolationStartTime,
+            lastUpdateTime: requestInvestigation.lastUpdateTime,
+            startTime: requestInvestigation.startTime,
+            symptomsStartTime: requestInvestigation.symptomsStartTime
+        }}).then((result: any) => {
```

Comment:
if you are not using the result than dont request it in the function, use only ()

Reply:
ok